src/node/__tests__/glob.test.ts

```diff
@@ -0,0 +1,110 @@
+import { Volume } from '../volume';
+
+describe('glob APIs', () => {
+  let vol: Volume;
+
+  beforeEach(() => {
+    vol = new Volume();
+    // Create a test file structure
+    vol.mkdirSync('/test', { recursive: true });
+    vol.mkdirSync('/test/subdir', { recursive: true });
+    vol.writeFileSync('/test/file1.js', 'console.log("file1");');
+    vol.writeFileSync('/test/file2.ts', 'console.log("file2");');
+    vol.writeFileSync('/test/file3.txt', 'hello world');
+    vol.writeFileSync('/test/subdir/nested.js', 'console.log("nested");');
+    vol.writeFileSync('/test/subdir/nested.py', 'print("nested")');
+  });
```

Comment:
Do not use `beforeEach()`, if you want to have a common setup code, wrap it in plain function:


```suggestion
const setup = () => {
    vol = new Volume();
    // Create a test file structure
    vol.mkdirSync('/test', { recursive: true });
    vol.mkdirSync('/test/subdir', { recursive: true });
    vol.writeFileSync('/test/file1.js', 'console.log("file1");');
    vol.writeFileSync('/test/file2.ts', 'console.log("file2");');
    vol.writeFileSync('/test/file3.txt', 'hello world');
    vol.writeFileSync('/test/subdir/nested.js', 'console.log("nested");');
    vol.writeFileSync('/test/subdir/nested.py', 'print("nested")');
    
    return {vol};
};
```